internal/util/util_test.go

```diff
@@ -0,0 +1,143 @@
+package util
+
+import (
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/operator-framework/rukpak/api/v1alpha1"
+	rukpakv1alpha1 "github.com/operator-framework/rukpak/api/v1alpha1"
+	plain "github.com/operator-framework/rukpak/internal/provisioner/plain/types"
+)
+
+func TestCheckDesiredBundleTemplate(t *testing.T) {
```

Comment:
Do we have a style guide for our repo's tests? I recently re-implemented the storage tests to use ginkgo/gomega thinking that's what we had more recently gravitated to. But maybe we haven't?

Reply:
Nope - I can refactor to use ginkgo as a follow-up? I use vscode locally which has some nice test autogeneration features, vs. having to bootstrap ginkgo style tests myself.

Reply:
I personally like ginkgo/gomega, but I don't think my opinion should be weighted as heavily as others. Do what folks think is most productive.